content/docs/start/experiments.md

```diff
@@ -18,17 +18,90 @@ once they're no longer needed.
 
 ## Running experiments
 
-Previously, we learned how to tune [ML pipelines](/doc/start/data-pipelines) and
-[compare the changes](/doc/start/metrics-parameters-plots). Let's further
-increase the number of features in the `featurize` stage to see how it compares.
+In the previous page, we learned how to tune
+[ML pipelines](/doc/start/data-pipelines) and
+[compare the changes](/doc/start/metrics-parameters-plots). In this section
+we'll use a
+[Get Started with Experiments project](https://github.com/iterative/get-started-experiments)
+to illustrate experimentation features in DVC 2.0.
+
+<details>
+
+### ⚙️ Click here for the installation instructions
+
+These commands are run in the
+[`get-started-experiments`](https://github.com/iterative/get-started-experiments)
+project. You can run these commands after cloning the repository and installing
+the requirements.
+
+Please clone and create a virtual environment:
```

Comment:
```suggestion
### ⚙️ Click here to setup the example project.

We'll use our
[`get-started-experiments`](https://github.com/iterative/get-started-experiments)
repo for the usage examples in this page. It can be obtained with Git:
```

Reply:
Done, thank you. 👍🏼